package.json

```diff
@@ -26,7 +25,8 @@
     "test": "TZ=UTC jest --coverage",
     "test:without-coverage": "TZ=UTC jest",
     "test:watch": "TZ=UTC jest --watch",
-    "test:snapshot:update": "TZ=UTC jest -u"
+    "test:snapshot:update": "TZ=UTC jest -u",
+    "prepare": "husky"
```

Comment:
Is this actually required in the latest Husky version? I slightly remember it is not used anymore. I don’t have in my private package, but Husky should work without preparation. 🤔

Reply:
this was automatically added by husky, not me